File: Azure Storage Account Key/storage_account_key.js
const { BlobServiceClient } = require('@azure/storage-blob');

const storageAccountName = 'your_storage_account_name';
const storageAccountKey = 'your_storage_account_key';
const containerName = 'your_container_name';

const connectionString = `DefaultEndpointsProtocol=https;AccountName=${storageAccountName};AccountKey=${storageAccountKey};EndpointSuffix=core.windows.net`;
const blobServiceClient = BlobServiceClient.fromConnectionString(connectionString);

async function listBlobs() {
    const containerClient = blobServiceClient.getContainerClient(containerName);
    let blobs = containerClient.listBlobsFlat();
    for await (const blob of blobs) {
        console.log(`Blob name: ${blob.name}`);
    }
}

listBlobs().catch(console.error);

File: Azure Storage Account Key/storage_account_key.py
from azure.storage.blob import BlobServiceClient

storage_account_name = "your_storage_account_name"
storage_account_key = "your_storage_account_key"
container_name = "your_container_name"

connection_string = f"DefaultEndpointsProtocol=https;AccountName={storage_account_name};AccountKey={storage_account_key};EndpointSuffix=core.windows.net"

blob_service_client = BlobServiceClient.from_connection_string(connection_string)

# List blobs in container
container_client = blob_service_client.get_container_client(container_name)
blob_list = container_client.list_blobs()
for blob in blob_list:
    print(blob.name)
